src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -39,15 +41,19 @@
 #endif
 #include <unistd.h>
 #include <pwd.h>
+#if !defined(_AIX)
```

Comment:
Why is this necessary?  If _AIX doesn't have these, shouldn't the cmake configuration just be setting HAVE_SENDFILE_4/6 to 0?